Helper/Requests.php

```diff
@@ -364,10 +364,7 @@ public function buildThreeDS2Data($additionalData, $storeId, $request = [])
      */
     public function buildRedirectData($storeId, $request = [])
```

Comment:
is this function used anywhere else than the redirectDataBuilder.php? Let's check it and if not then it can be removed.